.github/workflows/tests.yml

```diff
@@ -14,18 +14,15 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ 8, 11 ]
+        java: [ 17 ]
         experimental: [false]
-#        include:
-#          - java: 17
-#            experimental: true
       fail-fast: false
     continue-on-error: ${{ matrix.experimental }}
     name: Java ${{ matrix.Java }} build and test
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up java ${{ matrix.Java }}
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
```

Comment:
What does this change (v2 -> v3) do ?

Reply:
It stops github complaining that it's using deprecated stuff.